source/gtd/GirWrapper.d

```diff
@@ -585,56 +588,56 @@ class GirWrapper
 		}
 	}
 
-	private string getGirDirectory()
+	private string[] getGirDirectories()
 	{
-		version(Windows)
+		static string[] dirs;
+		if(dirs !is null)
 		{
-			import std.process : environment;
-
-			static string path;
+			return dirs;
+		}
+		else
+		{
+			dirs = [];
```

Comment:
This isn't necessary in D, you can just append to an empty array and the runtime will allocate it for you.
I would remove the whole else clause in this case.